client/Wayland/wlf_disp.c

```diff
@@ -250,9 +250,9 @@ void wlf_disp_free(wlfDispContext* disp)
 
 	if (disp->wlc)
 	{
-		PubSub_UnsubscribeActivated(disp->wlc->context.pubSub, wlf_disp_OnActivated);
-		PubSub_UnsubscribeGraphicsReset(disp->wlc->context.pubSub, wlf_disp_OnGraphicsReset);
-		PubSub_UnsubscribeTimer(disp->wlc->context.pubSub, wlf_disp_OnTimer);
+		PubSub_UnsubscribeActivated(disp->wlc->common.context.pubSub, wlf_disp_OnActivated);
```

Comment:
Perhaps a local variable aliasing `disp->wlc->common.pubSub` would make the code nicer...